assets/js/utils/attributes-query.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import { sortBy, map } from 'lodash';
+
+/**
+ * Given a query object, removes an attribute filter by a single slug.
+ * @param {object} query Current query object.
+ * @param {function} setQuery Callback to update the current query object.
+ * @param {object} attribute An attribute object.
```

Comment:
Correct notation for object type in jsdoc is `{Object}`.  There's a number of places where this needs fixed.  Same thing with `{Array}`.  Basically anything that is an object is Capitalized, primitives are lowercase - `Function` should be capitalized too (I keep forgetting on that one).